src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -537,31 +535,7 @@ public static GlobalSettings AddGlobalSettingsServices(this IServiceCollection s
     public static void UseDefaultMiddleware(this IApplicationBuilder app,
         IWebHostEnvironment env, GlobalSettings globalSettings)
     {
-        string GetHeaderValue(HttpContext httpContext, string header)
-        {
-            if (httpContext.Request.Headers.ContainsKey(header))
-            {
-                return httpContext.Request.Headers[header];
-            }
-            return null;
-        }
-
-        // Add version information to response headers
-        app.Use(async (httpContext, next) =>
-        {
-            using (LogContext.PushProperty("IPAddress", httpContext.GetIpAddress(globalSettings)))
-            using (LogContext.PushProperty("UserAgent", GetHeaderValue(httpContext, "user-agent")))
-            using (LogContext.PushProperty("DeviceType", GetHeaderValue(httpContext, "device-type")))
-            using (LogContext.PushProperty("Origin", GetHeaderValue(httpContext, "origin")))
-            {
-                httpContext.Response.OnStarting((state) =>
-                {
-                    httpContext.Response.Headers.Append("Server-Version", AssemblyHelpers.GetVersion());
```

Comment:
⚠️ We lost this as well -- is that handled elsewhere?

Reply:
No it's not, I could not see anything that was consuming it and we have the version on the `/config` endpoint anyway. Happy to add it back though.

Reply:
Worth asking all of engineering I think. Clients might be benefitting from it perhaps?

Reply:
No red alarms [brought up here](https://bitwarden.slack.com/archives/GRN5AAZ6Y/p1725659416925649). Want to put it's removal behind a feature flag so we can quickly turn it back on if anything goes wrong?